### ISSUE TITLE
Jailbroken Support (.deb)

### ISSUE THREAD

**Issue by CycloKid** [Pomelo-Emu/Pomelo#4]
Are there any plans to support jailbroken iDevices at some point in the future? It's my understanding that you can do a lot more with jailbreak debs than Trollstore or a paid developer account.

**Comment by TechGuy541** [comment]
Yup its on the way stay onboard the project, join the discord ill let u know as soon as it happens
